rhea/system/memmap/avalonmm.py

```diff
@@ -15,16 +15,18 @@
 
 
 class AvalonMM(MemoryMapped):
+
+    """AvalonMM bus object"""
+
     name = 'avalon'
 
     def __init__(self, glbl=None, data_width=8, address_width=16, name=None):
-        """ AvalonMM bus object
-        Parameters (kwargs):
-        --------------------
-        :param glbl: system clock and reset
-        :param data_width: data bus width
-        :param address_width: address bus width
-        :param name: name for the bus
+        """
```

Comment:
See previous comment. If we plan on using napoleon, this should be configured in sphinx.